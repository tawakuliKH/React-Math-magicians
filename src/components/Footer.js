const Footer = () => (
  <div className="navbar footer">
    <a to="/" className="nav-menu"> @copyright 2023</a>
  </div>
);
export default Footer;
