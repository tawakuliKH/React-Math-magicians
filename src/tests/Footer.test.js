import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

describe('Footer', () => {
  it('renders', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
