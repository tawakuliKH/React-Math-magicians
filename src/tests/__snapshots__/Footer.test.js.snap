// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders 1`] = `
<div
  className="navbar footer"
>
  <a
    className="nav-menu"
    to="/"
  >
     @copyright 2023
  </a>
</div>
`;
